app/commands/query_from_offset.go

```diff
@@ -18,10 +18,10 @@ import (
 
 // TODO(BlakeMScurr) usage info should be generated from cli
```

Comment:
Make sure you add this TODO as an issue, milestone, or in your project management tool; eg. Trello.

Reply:
+1 Thanks - it would be cool if you could do that for me, which would be interesting as the tenet would define the flow to a fairly large degree.

Reply:
Ideally, there would be a link to confirm in her comment.